firmware/application/apps/ui_scanner.cpp

```diff
@@ -536,7 +537,7 @@ ScannerView::ScannerView(
     receiver_model.set_squelch_level(0);
 
     // LOAD FREQUENCIES
-    frequency_file_load(get_freqman_path(freqman_file));
+    frequency_file_load(get_freqman_path(freqman_file, dir_profile::ProfileUser));
```

Comment:
In current firmware, the Scanner app loads the default file /FREQMAN/SCANNER.TXT, and allows new frequencies to be added to that.  With the proposed folder structure, the scanner app will not load any sample frequencies by default because you're only looking for the /USR/FREQMAN/SCANNER.TXT file here, and ignoring the sample frequencies file at /RES/FREQMAN/SCANNER.TXT.  I think if the /USR/FREQMAN/SCANNER.TXT file does not exist, I think the Scanner app should copy the file /RES/FREQMAN/SCANNER.TXT to the USR folder and then open it, so that the behavior will be the same as it was.

Reply:
Oh I’m sorry about this part, I didn’t notice. I will try to implement it in better way.

Reply:
fixed